app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/response/IncludedResponse.kt

```diff
@@ -0,0 +1,69 @@
+package com.kks.nimblesurveyjetpackcompose.model.response
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+enum class SurveyType(val typeValue: String) {
+    QUESTION("question"), ANSWER("answer")
+}
+
+sealed class IncludedResponse(@Json(name = "type") val type: String)
+
+@JsonClass(generateAdapter = true)
+data class IncludedQuestionResponse(
+    @Json(name = "id") val id: String? = null,
+    @Json(name = "attributes") val attributes: AttributeQuestionResponse? = null,
+    @Json(name = "relationships") val relationships: SurveyIncludedRelationshipsResponse? = null,
+) : IncludedResponse(SurveyType.QUESTION.typeValue) {
+    @JsonClass(generateAdapter = true)
```

Comment:
[Optional] Add a space here for readability, same down below 🤔 
```suggestion

        @JsonClass(generateAdapter = true)
```